server.js

```diff
@@ -49,6 +53,19 @@ function authenticate(code, cb) {
   req.on('error', function(e) { cb(e.message); });
 }
 
+function log(label, value, sanitized) {
```

Comment:
Add a JSDoc comment block to this function to document the function, the parameters, what each parameter is for, and what types of data are expected for each parameter.